packages/integrations/mdx/README.md

```diff
@@ -306,55 +306,102 @@ export default {
 
 ### remarkPlugins
 
-**Default plugins:** [remark-gfm](https://github.com/remarkjs/remark-gfm), [remark-smartypants](https://github.com/silvenon/remark-smartypants)
-
 [Remark plugins](https://github.com/remarkjs/remark/blob/main/doc/plugins.md) allow you to extend your Markdown with new capabilities. This includes [auto-generating a table of contents](https://github.com/remarkjs/remark-toc), [applying accessible emoji labels](https://github.com/florianeckerstorfer/remark-a11y-emoji), and more. We encourage you to browse [awesome-remark](https://github.com/remarkjs/awesome-remark) for a full curated list!
 
-We apply [GitHub-flavored Markdown](https://github.com/remarkjs/remark-gfm) and [Smartypants](https://github.com/silvenon/remark-smartypants) by default. This brings some niceties like auto-generating clickable links from text (ex. `https://example.com`) and formatting quotes for readability. When applying your own plugins, you can choose to preserve or remove these defaults.
-
-To apply plugins _while preserving_ Astro's default plugins, use a nested `extends` object like so:
+This example applies the [`remark-toc`](https://github.com/remarkjs/remark-toc) plugin to `.mdx` files. To customize plugin inheritance from your Markdown config or Astro's defaults, [see the `extendPlugins` option](#extendPlugins).
 
 ```js
 // astro.config.mjs
 import remarkToc from 'remark-toc';
 
 export default {
   integrations: [mdx({
-    // apply remark-toc alongside GitHub-flavored markdown and Smartypants
-    remarkPlugins: { extends: [remarkToc] },
+    remarkPlugins: [remarkToc],
   })],
 }
 ```
 
-To apply plugins _without_ Astro's defaults, you can apply a plain array:
+### rehypePlugins
+
+[Rehype plugins](https://github.com/rehypejs/rehype/blob/main/doc/plugins.md) allow you to transform the HTML that your Markdown generates. We encourage you to browse [awesome-rehype](https://github.com/rehypejs/awesome-rehype) for a full curated list of plugins!
+
+We apply our own (non-removable) [`collect-headings`](https://github.com/withastro/astro/blob/main/packages/integrations/mdx/src/rehype-collect-headings.ts) plugin. This applies IDs to all headings (i.e. `h1 -> h6`) in your MDX files to [link to headings via anchor tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_an_element_on_the_same_page).
+
+This example applies the [`rehype-minify`](https://github.com/rehypejs/rehype-minify) plugin to `.mdx` files. To customize plugin inheritance from your Markdown config or Astro's defaults, [see the `extendPlugins` option](#extendPlugins).
 
 ```js
 // astro.config.mjs
-import remarkToc from 'remark-toc';
+import rehypeMinifyHtml from 'rehype-minify';
 
 export default {
   integrations: [mdx({
-    // apply remark-toc alone, removing other defaults
+    rehypePlugins: [rehypeMinifyHtml],
+  })],
+}
+```
+
+### extendPlugins
+
+**Type:** `'markdown' | 'defaults' | false`
+
+**Default:** `'markdown'`
+
+#### `markdown` (default)
```

Comment:
"This is more of a comment than a question..." I'm just noting that the default setting is `markdown` while `defaults` is *not* the default setting. Do with that observation what you will.

Reply:
Yep, it's a tough one! Not sure what else to name Astro's default plugins other than... `defaults`. Suggestions welcome.

Reply:
Update: changed to `astroDefaults` for a little more clarity!